extensions/dataseed/dataseed-atlas/src/main/java/com/microsoft/dagx/dataseed/atlas/AtlasDataSeederExtension.java

```diff
@@ -29,7 +30,7 @@
     private AtlasDataSeeder dataSeeder;
 
     public AtlasDataSeederExtension() {
```

Comment:
I'm currently thinking whether this is more like an atlas demo extension. Because (if I understand the code correct) it only works with Azure.